### ISSUE TITLE
Feature Request: remove the 100MB limit for demo sites, even if it is payed

### ISSUE THREAD

**Issue by LauDijksterhuis** [Automattic/studio#482]
### What

I have been working on a website for my client in WordPress Studio, which I really enjoyed.
So I wanted to publish a demo site but then I got the notification, 100MB limit exceeded.

I do understand that you can not give out free hosting to everyone that uses the WordPress Studio app.
But it would be nice if we can at least pay for the option to remove the limit.

Or even better, connect to a hosting plan as mentioned here: https://github.com/Automattic/studio/issues/62 

![CleanShot 2024-08-20 at 09 29 58](https://github.com/user-attachments/assets/83f70361-7a67-413d-b74f-fb67ee16322f)

### Why

Well, because I have been working a lot on this project and it is kinda frustrating that suddenly the demo site option doesn't work anymore... I have used it in the very beginning so I was expecting it to work just fine

But since it is not mentioned anywhere inside the app its a real bummer...

### How

I already gave some thoughts on connecting Studio to WordPress.com hosting plan in the following issue: 
https://github.com/Automattic/studio/issues/62 

However, I can imagine other people/agencies wanting to sync with their own hosting provider
So maybe with a FTP/SSH connection there other possibilities to sync with any preferred provider.

**Comment by katinthehatsite** [comment]
@LauDijksterhuis Thanks for your feedback!

A bit of news on this is that in the next app release, the space limit will be increased to 250MB: https://github.com/Automattic/studio/pull/446

**Comment by LauDijksterhuis** [comment]
Hi @katinthehatsite ,

Ah, that sounds great, however I do want to stress the importance of being able to move the site to a live environment.
The documentation https://developer.wordpress.com/docs/developer-tools/studio/ states that the recommended way of migrating a site to a host is via the All in one WP migration plugin.

![CleanShot 2024-08-20 at 10 20 43](https://github.com/user-attachments/assets/62313359-2ca4-4393-bd1f-f6168f641b5e)

I just tried to move the site to WordPress.com to show to my client via this plugin but it as you can see it didn't work wel...
(I have blurred the screenshots as this website is still in development, but you can see the big white areas around the picture of the main page)

Localhost:
![CleanShot 2024-08-20 at 09 41 31](https://github.com/user-attachments/assets/249c9d30-f3d5-4bcd-a567-70bb6f636544)

Live site:
![CleanShot 2024-08-20 at 09 41 36](https://github.com/user-attachments/assets/f601e7ff-6263-458e-9d32-0b4d7c414d08)


I know this is a fault in the plugin and not specifically something WordPress team can fix.
But without a proper way of migrating the site to a host the whole idea of developing locally loses its meaning.

In the past I used the WPVivid plugin to backup and migrate a site, but this option is not possible in Studio. 
Due to "The curl extension is not detected. Please install the extension first." 

Next I am going to try the Migrate Guru plugin, which is recommended somewhere else in the WordPress.com documentation, but I feel like WordPress.com shouldn't rely on third party plugins for something this important

Especially when Automattic has a backup solution (Jetpack) build in...

**Comment by LauDijksterhuis** [comment]
Update, Migrate Guru also doesn't seem to work...

**Comment by LauDijksterhuis** [comment]
Update: Migrate Guru doesn't work as the website is not live...

**Comment by wojtekn** [comment]
@LauDijksterhuis we are working on the Import / Export possibilities. Feel free to test the beta: [Mac Intel (DMG)](https://href.li/?https://cdn.a8c-ci.services/studio/studio-darwin-x64-v1.1.0-beta4.dmg) or [Mac Apple Silicon (DMG)](https://href.li/?https://cdn.a8c-ci.services/studio/studio-darwin-arm64-v1.1.0-beta4.dmg).

**Comment by LauDijksterhuis** [comment]
Hi @wojtekn 
Thank you for the beta access, I did manage to make an export and use this guide: https://jetpack.com/support/backup/manually-restore-a-backup/ 
To get a working version online!

It does feel a bit hacky and not as streamlined, and I am sure the Automattic team is working on some other way of importing the WordPress export/backup. But do you have some information on when/how/where I can find more information about this?

I really look forward to implementing Studio in my workflow :)

**Comment by wojtekn** [comment]
> Thank you for the beta access, I did manage to make an export and use this guide: https://jetpack.com/support/backup/manually-restore-a-backup/
> To get a working version online!

I'm glad it worked for you!

> It does feel a bit hacky and not as streamlined, and I am sure the Automattic team is working on some other way of importing the WordPress export/backup. But do you have some information on when/how/where I can find more information about this?

We are considering adding such a feature, but we don't have a timeline yet.

**Comment by LauDijksterhuis** [comment]
Hi @katinthehatsite 
For my project 250MB is still not enough sadly, but I do understand that some limit is required to prevent having to much resources used on the demosite servers.
So I will be closing this issue as I think the solution is probably in the import/export functionality in Jetpack: https://github.com/Automattic/jetpack/issues/38987 

Or preferably in the sync with hosting plan as mentioned here: https://github.com/Automattic/studio/issues/62

**Comment by danielbachhuber** [comment]
@LauDijksterhuis Out of curiosity, how large is your site?

**Comment by LauDijksterhuis** [comment]
Hi @danielbachhuber 

Not sure if I should count the MB's of the whole folder, but at this moment it is almost 760MB and growing...
My client already has a WordPress website at this moment which is almost 7GB.
I do think a lot of the old site is unnecessary, that's why I started from scratch.

However the current project (760MB) is just a first impression to show to my client and will, when finished, probably still be around 3GB. 

So for now I have purchased a WordPress.com creator plan and I am continuing development on the server instead of locally.

![CleanShot 2024-08-21 at 13 04 20](https://github.com/user-attachments/assets/ab02b305-d023-48e0-a8b5-effcd1142d5c)
